### DISCUSSION TITLE
source/PlayState.hx:1881: characters 26-27 : Unexpected ,

### DISCUSSION THREAD

**Discussion by les-citrons** [FunkinCrew/Funkin#1484] [Compiling help]
I am supplied with the following error when trying to build the game:
```
source/PlayState.hx:1881: characters 26-27 : Unexpected ,
source/PlayState.hx:1881: characters 26-27 : Unexpected ,
source/PlayState.hx:1881: characters 26-27 : Unexpected ,
source/PlayState.hx:1881: characters 26-27 : Unexpected ,
source/PlayState.hx:1881: characters 26-27 : Unexpected ,
```

**Comment by SilverT4** [comment]
try checking for any extra/missing double quotes, curly brackets {}, parenthesis, etc. it's possible you have an extra character somewhere.

**Comment by les-citrons** [comment]
That would be odd, considering I haven't modified that file. I will try to check.

**Comment by Epicminer256** [comment]
Update your newsground with haxelib update

**Comment by les-citrons** [comment]
`haxelib update newgrounds`
`newgrounds is up to date`

**Comment by Senji72** [comment]
Thanks for this information
